src/backend/distributed/master/master_truncate.c

```diff
@@ -73,7 +73,14 @@ citus_truncate_trigger(PG_FUNCTION_ARGS)
 	{
 		List *taskList = TruncateTaskList(relationId);
 
-		ExecuteUtilityTaskListWithoutResults(taskList);
+		/*
+		 * If it is a local placement of a distributed table, then execute
+		 * the TRUNCATE command locally.
+		 */
+		bool localExecutionSupported = true;
+
+		/* use adaptive executor when enabled */
```

Comment:
I don't think the adaptive executor can be disabled at this point.

Reply:
I guess it's there due to copy-paste things :)

Reply:
I will remove it

Reply:
done